worlds/tunic/__init__.py

```diff
@@ -332,8 +371,75 @@ def remove_filler(amount: int) -> None:
             if tunic_item.name in slot_data_item_names:
                 self.slot_data_items.append(tunic_item)
 
+        # pull out the filler so that we can place it manually during pre_fill
+        self.grass_fill = []
+        self.grass_fill_locations = []
+        self.non_grass_fill = []
+        self.non_grass_fill_locations = []
+        # grass and non-grass get their own pools
+        if self.options.grass_randomizer:
+            filler = self.grass_fill
+            fill_locations = self.grass_fill_locations
+        else:
+            filler = self.non_grass_fill
+            fill_locations = self.non_grass_fill_locations
```

Comment:
I don't love this, but I'll let it slide :D

*comment*

Reply:
Ftr, what I'd prefer is if at this stage, it was just `self.filler` and `self.fill_locations`, and you checked which list to extend in `stage_prefill` by checking `world.options.grass_rando` in the `for world in tunic_fill_worlds:` for loop.

Reply:
This one is not blocking tho, I can let this slide

Reply:
Resolved this in a PR